walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,49 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    //mandatory attendees of meeting request
+    Collection<String> attendees = request.getAttendees();
```

Comment:
I'd like to challenge you to use multiple helper methods here instead of having this one long method. Having helper methods can help make the code more readable as it encompasses related code together and has a helpful name that can give the user an idea as to what is being done, regardless of whether or not they want to actually read the code in that helper method.